makebeta

```diff
@@ -49,7 +49,23 @@ echo "Success, making beta...";
 cp -f appversion.h ${version_file}
 cp -f History.txt ${history_file}
 
-# Generate the archive
+##############################################
+# Cleaning JavaScript files in directory WWW #
+##############################################
+
+# Remove all source .js files as they should now be compressed
+find www -name *.js -exec rm '{}' \;
+
+# Remove all intermediate files ending in .min
+find www -name *.min -exec rm '{}' \;
+
+# Rename files from *.js.min.gz to *.js.gz before inclusion to archive
```

Comment:
A HTML designer is good with HTML, not with c++, does not even know how to install visual studio, or gcc, let alone compile or knows the Linux or Windows operating system.
That is why minification is not really usefull.  Specially not for the domoticz related stuff, sure for third party libraries, but there are for 90% already minimized

But great stuff !, Definitely going to look into this before a new stable release !